test/e2e/framework/service_util.go

```diff
@@ -490,13 +501,18 @@ func (j *ServiceTestJig) newRCTemplate(namespace string) *v1.ReplicationControll
 			Labels:    j.Labels,
 		},
 		Spec: v1.ReplicationControllerSpec{
-			Replicas: func(i int) *int32 { x := int32(i); return &x }(1),
+			Replicas: func(i int) *int32 { x := int32(i); return &x }(2),
 			Selector: j.Labels,
 			Template: &v1.PodTemplateSpec{
 				ObjectMeta: metav1.ObjectMeta{
 					Labels: j.Labels,
 				},
 				Spec: v1.PodSpec{
+					Affinity: &v1.Affinity{
```

Comment:
Done.  @krousey can you take a look at how I structured it?  I used the existing tweak hook, but the implementation is a little ugly.